systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -325,8 +335,9 @@
         end
         if body_id == loading_foot;
           foot_actual = obj.robot.forwardKin(kinsol, qp_input.body_motion_data(j).body_id, [0;0;0], 0);
-          foot_des = evalCubicSplineSegment(t_global - qp_input.body_motion_data(j).ts(1), qp_input.body_motion_data(j).coefs);
-          obj.plan_shift(1:3) = foot_des(1:3) - foot_actual(1:3);
+          % apply plan shift based on nearest-in-time coefs
+          foot_des = evalCubicSplineSegment(t_global - qp_input.body_motion_data(j).ts(1), qp_input.body_motion_data(j).coefs(:, 1, :));
+          obj.plan_shift(1:3, :) = foot_des(1:3) - foot_actual(1:3);
```

Comment:
we're deprecating that whole matlab file, so we should probably just lose this change